.circleci/config.yml

```
@@ -43,6 +43,9 @@ jobs:
     steps:
       - attach_workspace:
           at: ./
+      - run:
+          name: Authenticate with NPM registry
+          command: echo '<//npm.jahia.com/:_authToken=>$NPM_TOKEN' >>  `npm config get userconfig`
```

what are this < > for ??
```suggestion
          command: echo '//npm.jahia.com/:_authToken=$NPM_TOKEN' >>  `npm config get userconfig`
```

---

I've pasted the wrong command. Thank you.